xapian-applications/omega/configure.ac

```diff
@@ -246,6 +246,16 @@ AC_SEARCH_LIBS([socketpair], [xnet],
 dnl fnmatch() is used to match ignore pattern.
 AC_CHECK_FUNCS([fnmatch])
 
+dnl check if poppler is available.
+am_save_LIBS="$LIBS"
+AC_LIB_APPENDTOVAR([LIBS], [-lpoppler-cpp])
+AC_TRY_LINK([#include <poppler/cpp/poppler-version.h>],
+            [poppler::version_string()],
+            [AC_DEFINE(HAVE_POPPLER, 1, [Define HAVE_POPPLER if the labrary is available]), lib_poppler=true],
+	    [lib_poppler=false])
+AM_CONDITIONAL([OMINDEX_PDF], [test x$lib_poppler != xfalse])
+LIBS="$am_save_LIBS"
```

Comment:
I have not much experience with this kind of tools, Did I modify it correctly?
It seems to compile on Travis. I also add a case to travis (similar to the first one, in a personal repository) where poppler is not available and it passed.